python/pylibcudf/pylibcudf/tests/test_scalar.py

```diff
@@ -28,3 +28,46 @@ def test_from_py_notimplemented(val):
 def test_from_py_typeerror(val):
     with pytest.raises(TypeError):
         plc.Scalar.from_py(val)
+
+
+@pytest.mark.parametrize(
+    "np_type",
+    [
+        "bool_",
+        "str_",
+        "int8",
+        "int16",
+        "int32",
+        "int64",
+        "uint8",
+        "uint16",
+        "uint32",
+        "uint64",
+        "float32",
+        "float64",
+    ],
+)
+def test_from_numpy(np_type):
+    np = pytest.importorskip("numpy")
```

Comment:
Depending on how many times you anticipate this popping up, you can make a fixture for numpy itself:
```
@pytest.fixture(scope="module")
def np():
    return pytest.importorskip("numpy")
```
Not required, just a note.